lib/hotkeys.ts

```diff
@@ -0,0 +1,29 @@
+const NON_ENGLISH_LAYOUT = /^[^\x00-\x7F]$/
+
+function compareHotkey(hotkey: string, e: KeyboardEvent): boolean {
+  let prefix = ''
+  if (e.metaKey) prefix += 'meta+'
+  if (e.ctrlKey) prefix += 'ctrl+'
+  if (e.altKey) prefix += 'alt+'
+  if (e.shiftKey) prefix += 'shift+'
+
+  let code = prefix
+  if (e.key === '+') {
+    code += 'plus'
+  } else {
+    code += e.key.toLowerCase()
+  }
+
+  let isMatch = code === hotkey
+
+  if (!isMatch && NON_ENGLISH_LAYOUT.test(e.key) && /^Key.$/.test(e.code)) {
+    let enKey = e.code.replace(/^Key/, '').toLowerCase()
```

Comment:
English fix is nice, but now we have code duplication between `view/fields` and here. 

They use hotkeys too.

Reply:
moved this logic to separate function in lib/hotkeys. I assumed that we don't use non english letters as hotkeys. so I convert all non english letters, and after that check if hotkey matches